PerilEndorsements.py

```diff
@@ -0,0 +1,102 @@
+import requests
+import json
+
+def perilEndorsement():
+    print "Getting authorization token."
+
+    #URL = "https://api.develop.socotra.com/account/authenticate"
+    URL = "http://127.0.0.1:8080/account/authenticate"
+    #URL = "https://api.develop.socotra.com/account/v1/authenticateInternal
+    payload = {
+                "username": "alice.lee",
+                "password": "socotra",
+                "hostName": "vishnu-vishnu1.co.develop.socotra.com"
+                }
+
+    r = requests.post(URL, json=payload)
+    data = json.loads(r.text)
+    authorizationToken = data["authorizationToken"]
```

Comment:
You don't need the json import above. You can do this:
```
    auth_token = r.json()['authorizationToken']
```
For reference: http://docs.python-requests.org/en/master/user/quickstart/#json-response-content